src/content/en/2019/page-weight.md

```diff
@@ -6,809 +6,765 @@ description: Page Weight chapter of the 2019 Web Almanac covering why page weigh
+#### Mobile
 
+<figure>
+  <table>
+    <tr>
+      <th>Percentile</th>
+      <th>Total (KB)</th>
+      <th>HTML (KB)</th>
+      <th>JS (KB)</th>
+      <th>CSS (KB)</th>
+      <th>Image (KB)</th>
+      <th>Document (KB)</th>
+    </tr>
+    <tr>
+      <td>90</td>
+      <td>+376</td>
+      <td>-50</td>
+      <td>+46</td>
+      <td>+36</td>
+      <td>+648</td>
+      <td>+2</td>
+    </tr>
+    <tr>
+      <td>75</td>
+      <td>+304</td>
+      <td>-7</td>
+      <td>+34</td>
+      <td>+21</td>
+      <td>+281</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <td>50</td>
+      <td>+179</td>
+      <td>-1</td>
+      <td>+27</td>
+      <td>+10</td>
+      <td>+106</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <td>25</td>
+      <td>+110</td>
+      <td>-1</td>
+      <td>+16</td>
+      <td>+5</td>
+      <td>+36</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <td>10</td>
+      <td>+72</td>
+      <td>0</td>
+      <td>+13</td>
+      <td>+2</td>
+      <td>+20</td>
+      <td>+1</td>
+    </tr>
+  </table>
+  <figcaption>Figure 5. Change (KB) in page size vs. 2018</figcaption>
+</figure>
 
-## Page Weight
+#### Desktop
 
-Roughly speaking, mobile sites are about 10% smaller than their desktop counterparts. The majority of the difference is due to mobile sites loading less image bytes than their desktop counterparts.
+<figure>
+  <table>
+    <tr>
+      <th>Percentile</th>
+      <th>Total (KB)</th>
+      <th>HTML (KB)</th>
+      <th>JS (KB)</th>
+      <th>CSS (KB)</th>
+      <th>Image (KB)</th>
+      <th>Document (KB)</th>
+    </tr>
+    <tr>
+      <td>90</td>
+      <td>+1106</td>
+      <td>-75</td>
+      <td>+22</td>
+      <td>+45</td>
+      <td>+1291</td>
+      <td>+5</td>
+    </tr>
+    <tr>
+      <td>75</td>
+      <td>+795</td>
+      <td>-12</td>
+      <td>+9</td>
+      <td>+32</td>
+      <td>+686</td>
+      <td>+1</td>
+    </tr>
+    <tr>
+      <td>50</td>
+      <td>+434</td>
+      <td>-1</td>
+      <td>+10</td>
+      <td>+15</td>
+      <td>+336</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <td>25</td>
+      <td>+237</td>
+      <td>0</td>
+      <td>+12</td>
+      <td>+7</td>
+      <td>+138</td>
+      <td>0</td>
+    </tr>
+    <tr>
+      <td>10</td>
+      <td>+120</td>
+      <td>0</td>
+      <td>+10</td>
+      <td>+2</td>
+      <td>+39</td>
+      <td>+1</td>
+    </tr>
+  </table>
+  <figcaption>Figure 6. Change (KB) in page size vs. 2018</figcaption>
+</figure>
+
+For a longer-term perspective on how page weight has changed over time, check out this [graph](https://httparchive.org/reports/page-weight#bytesTotal) on the main HTTP Archive site. Median page size has grown at a fairly constant rate since the HTTP Archive started tracking this metric in November 2010 and the increase in page weight observed over the past year is consistent with this.
```

Comment:
```suggestion
For a longer-term perspective on how page weight has changed over time, check out [this timeseries graph](https://httparchive.org/reports/page-weight#bytesTotal) from HTTP Archive. Median page size has grown at a fairly constant rate since the HTTP Archive started tracking this metric in November 2010 and the increase in page weight observed over the past year is consistent with this.
```